src/server/game/Spells/SpellMgr.cpp

```diff
@@ -2637,12 +2637,31 @@ void SpellMgr::LoadSpellInfoCustomAttributes()
 
                     if ((attributes & (SPELL_ATTR0_CU_NEGATIVE_EFF0 << i)) != 0)
                     {
-                        TC_LOG_ERROR("sql.sql", "Table `spell_custom_attr` has attribute SPELL_ATTR0_CU_NEGATIVE_EFF%u for spell %u with no EFFECT_%u", uint32(i), spellId, uint32(i));
+                        TC_LOG_ERROR("sql.sql", "Table `spell_custom_attr` has attribute SPELL_ATTR0_CU_NEGATIVE_EFF%u for spell %u with no EFFECT_%u, ignored.", uint32(i), spellId, uint32(i));
                         continue;
                     }
                 }
             }
 
+            if ((attributes & SPELL_ATTR0_CU_DONT_RESET_PERIODIC_TIMER) != 0)
+            {
+                bool hasAuraEffect = false;
+                for (uint8 i = 0; i < MAX_SPELL_EFFECTS; ++i)
+                {
+                    if (spellInfo->Effects[i].IsAura())
+                    {
+                        hasAuraEffect = true;
+                        break;
+                    }
+                }
+
+                if (!hasAuraEffect)
+                {
+                    TC_LOG_ERROR("sql.sql", "Table `spell_custom_attr` has attribute SPELL_ATTR0_CU_DONT_RESET_PERIODIC_TIMER for spell %u but it has no aura effect, ignored.", spellId);
```

Comment:
@illfated engrish is correct? kappa

Reply:
Looks good enough to me, although you could add a comma between "%u" and "but" if you are going to edit the file later. :)
`"[...]for spell %u, but it has no aura effect, ignored.`
Thanks for asking. I ❤️ correcting English grammar. 😉